gnd/src/main/java/com/google/android/gnd/ui/common/photoview/PhotoViewViewModel.java

```diff
@@ -0,0 +1,52 @@
+package com.google.android.gnd.ui.common.photoview;
+
+import android.net.Uri;
+import android.view.View;
+import androidx.databinding.ObservableInt;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import com.google.android.gnd.model.form.Field;
+import com.google.android.gnd.model.observation.Response;
+import com.google.android.gnd.system.StorageManager;
+import com.google.android.gnd.ui.common.AbstractViewModel;
+import javax.inject.Inject;
+
+public class PhotoViewViewModel extends AbstractViewModel {
+
+  private final StorageManager storageManager;
+  private final MutableLiveData<Uri> destinationPath = new MutableLiveData<Uri>();
+  private final ObservableInt isDestinationPathAvailable = new ObservableInt(View.GONE);
```

Comment:
This should probably be calling this "visibility" instead given the value it returns and how it's used.

Reply:
Also, we should use `LiveData` since `ObservableInt` isn't Lifecycle aware.